src/Carousel.vue

```diff
@@ -177,7 +177,7 @@ export default {
      * Listen for an external navigation request using this prop.
      */
     navigateTo: {
-      type: Number,
+      type: [Number, Array],
```

Comment:
can't we do 
```json
type: Number | [Number, Array],
```

Reply:
Hi Adin, could you explain what this would do / improve?